Gems/Atom/RPI/Assets/ShaderLib/Atom/RPI/ShaderResourceGroups/DefaultSceneSrg.azsli

```diff
@@ -13,6 +13,5 @@
 partial ShaderResourceGroup SceneSrg
 {
     float m_time;
-    float m_deltaTime;
```

Comment:
Out of curiosity, why was this field removed? I didn't see it mentioned in the description and it seems pretty useful.

Reply:
I think m_deltaTime is not well defined constant to use. 
Assume you have a Scene with 2 render pipelines, and each of the render pipeline has its own preferred render frequency. 
In some frame, one render pipeline is rendered but not the other one. For the scene's m_deltaTime, it would be updated each frame. For each render pipeline, they won't get the deltaTime for the frames which it wasn't rendered. 

It's more safe and accurate to save the last time in GPU buffer and use currentTime - lastTime to estimate deltaTime.